Quick_Deploy/HuggingFaceTransformers/falcon7b/config.pbtxt

```diff
@@ -0,0 +1,40 @@
+# Triton backend to use
+backend: "python"
+
+# Hugging face model path
+# Parameters must follow this
+# key/value structure
```

Comment:
The formatting is a bit weird/inconsistent in this file:
1. Comments are weirdly narrow - can probably fit on 1 or 2 lines
2. Parameters a different indentation than I/O

Reply:
1. I was trying to pad my contributed lines stat 😉. Jk fixed,
2. Fixed.